test/check/check.go

```diff
@@ -359,6 +376,14 @@ func checkForDaemonSet(name string) error {
 	return nil
 }
 
+func checkForSecret(name string) error {
+	return framework.Global.Client.Get(context.Background(), types.NamespacedName{Name: name}, &corev1.Secret{})
```

Comment:
As Quique commented offline, this and the following function misses namespace definition.

Reply:
done